geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -65,19 +71,21 @@
 import org.apache.geode.management.internal.configuration.validators.MemberValidator;
 import org.apache.geode.management.internal.configuration.validators.RegionConfigValidator;
 import org.apache.geode.management.internal.exceptions.EntityNotFoundException;
+import org.apache.geode.management.internal.operation.AsyncExecutorManager;
 import org.apache.geode.management.runtime.RuntimeInfo;
 
 public class LocatorClusterManagementService implements ClusterManagementService {
   private static final Logger logger = LogService.getLogger();
   private ConfigurationPersistenceService persistenceService;
   private Map<Class, ConfigurationManager> managers;
   private Map<Class, ConfigurationValidator> validators;
+  private AsyncExecutorManager executorManager;
   private MemberValidator memberValidator;
   private CacheElementValidator commonValidator;
 
   public LocatorClusterManagementService(InternalCache cache,
       ConfigurationPersistenceService persistenceService) {
-    this(persistenceService, new HashMap<>(), new HashMap<>(), null, null);
+    this(persistenceService, new HashMap<>(), new HashMap<>(), null, null, null);
```

Comment:
After this constructor completes are these HashMaps ever modified? If not it seems like this code should be refactored to fill the maps before passing them in. The other constructor that takes the Map params could then wrap then with Collections.unmodifiableMap.
If they can be modified later then we need to address thread-safety on them.

Reply:
the known ones are hardcoded for now, but extensions could register more so we don't want to make them immutable.  down the road when Geode is more modular, each module would register its own configurables and operations.